packages/opentelemetry-context-zone-peer-dep/package.json

```diff
@@ -79,12 +80,12 @@
     "typescript": "4.4.4",
     "webpack": "4.46.0",
     "webpack-cli": "4.10.0",
-    "zone.js": "0.11.4"
+    "zone.js": "0.13.3"
   },
   "peerDependencies": {
     "@opentelemetry/api": ">=1.0.0 <1.8.0",
-    "zone.js": "^0.10.2 || ^0.11.0"
+    "zone.js": "^0.10.2 || ^0.11.0 || ^0.13.0"
```

Comment:
In the meanwhile, Angular 17 (with zone.js 0.14) is out there.
Do you think would make sense to include that change here, since it is still ongoing?